src/Loggers/OwnSplitChannel.cpp

```diff
@@ -18,9 +18,6 @@ namespace DB
 
 void OwnSplitChannel::log(const Poco::Message & msg)
 {
-    if (!isLoggingEnabled())
```

Comment:
May I ask why this was removed? We still call `disableLogging` in `~BaseDaemon()`.

Reply:
I used it initially for text logs, but I defined a new object later on.
I'll return the code but I think OwnSplitChannel is unused now.